src/components/EventDetails.scss

```diff
@@ -14,8 +14,10 @@
   flex: 1;
 }
 .event-details__column--sidebar {
+  @media screen and (min-width: 500px) {
   flex: 0 0 360px;
   margin-left: 40px;
```

Comment:
indent?

Reply:
hmmm.. this is indented locally, taking a look